scaffolds/scala/src/main/scala/euranova/main.scala

```diff
@@ -0,0 +1,10 @@
+package euranova
+
+object ProjectMain {
```

Comment:
It's good practice to name your object the same as the .scala file. Likewise, it's good practice to start your file names with a capital letter.